03.bowling/bowling.rb

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+scores.each do |s|
+  if s == 'X'
+    shots << 10
+    shots << 0
+  else
+    shots << s.to_i
+  end
+end
+
+frames = shots.each_slice(2).to_a
+
+point = 0
+frames.each_with_index do |frame, i|
```

Comment:
```suggestion
frames.each_with_index.sum do |frame, i|
```

のようにすれば `point +=` の加算が不要になりそうです

Reply:
なるほど！ありがとうございます。